icu4j/main/classes/core/src/com/ibm/icu/text/BreakIteratorFactory.java

```diff
@@ -151,7 +157,7 @@ private static BreakIterator createBreakInstance(ULocale locale, int kind) {
         // Create a normal RuleBasedBreakIterator.
         //
         try {
-            iter = RuleBasedBreakIterator.getInstanceFromCompiledRules(bytes);
+            iter = RuleBasedBreakIterator.getInstanceFromCompiledRules(bytes, brkfname.startsWith("line_phrase_cj"));
```

Comment:
If in the previous try {} catch (Exception e) there are missing resources, then brkfname will be null, right? In that case brkfname will be null here.

Reply:
Resolved